tests/probly/transformation/evidential/regression/test_torch.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import pytest
+
+from probly.transformation.evidential.regression import evidential_regression
+from tests.probly.torch_utils import count_layers
+from probly.layers.torch import NormalInverseGammaLinear
+
+torch = pytest.importorskip("torch")
+
+from torch import nn  # noqa: E402
```

Comment:
Please make sure that all Ruff warnings are addressed.

```suggestion
"""TODO!"""

from __future__ import annotations

import pytest

from probly.layers.torch import NormalInverseGammaLinear
from probly.transformation.evidential.regression import evidential_regression
from tests.probly.torch_utils import count_layers

torch = pytest.importorskip("torch")

from torch import nn  # noqa: E402
```